packages/ember-toucan-form/src/-private/textarea-field.hbs

```diff
@@ -18,66 +18,174 @@
   a string is easy.
 }}
 <@form.Field @name={{@name}} as |field|>
-  {{#if (this.hasOnlyLabelBlock (has-block 'label') (has-block 'hint'))}}
-    <Form::Fields::Textarea
-      @hint={{@hint}}
-      @error={{this.mapErrors field.rawErrors}}
-      @value={{this.assertString field.value}}
-      {{! @glint-expect-error }}
-      @onChange={{field.setValue}}
-      @isDisabled={{@isDisabled}}
-      @isReadOnly={{@isReadOnly}}
-      @rootTestSelector={{@rootTestSelector}}
-      name={{@name}}
-      ...attributes
-    >
-      <:label>{{yield to='label'}}</:label>
-    </Form::Fields::Textarea>
-  {{else if (this.hasHintAndLabelBlocks (has-block 'label') (has-block 'hint'))
-  }}
-    <Form::Fields::Textarea
-      @error={{this.mapErrors field.rawErrors}}
-      @value={{this.assertString field.value}}
-      {{! @glint-expect-error }}
-      @onChange={{field.setValue}}
-      @isDisabled={{@isDisabled}}
-      @isReadOnly={{@isReadOnly}}
-      @rootTestSelector={{@rootTestSelector}}
-      name={{@name}}
-      ...attributes
-    >
-      <:label>{{yield to='label'}}</:label>
-      <:hint>{{yield to='hint'}}</:hint>
-    </Form::Fields::Textarea>
-  {{else if (this.hasLabelArgAndHintBlock @label (has-block 'hint'))}}
-    <Form::Fields::Textarea
-      @label={{@label}}
-      @error={{this.mapErrors field.rawErrors}}
-      @value={{this.assertString field.value}}
-      {{! @glint-expect-error }}
-      @onChange={{field.setValue}}
-      @isDisabled={{@isDisabled}}
-      @isReadOnly={{@isReadOnly}}
-      @rootTestSelector={{@rootTestSelector}}
-      name={{@name}}
-      ...attributes
-    >
-      <:hint>{{yield to='hint'}}</:hint>
-    </Form::Fields::Textarea>
+  {{#if (has-block 'secondary')}}
+    {{#if (this.hasOnlyLabelBlock (has-block 'label') (has-block 'hint'))}}
+      <Form::Fields::Textarea
+        @hint={{@hint}}
+        @error={{this.mapErrors field.rawErrors}}
+        @value={{this.assertString field.value}}
+        {{! @glint-expect-error }}
+        @onChange={{field.setValue}}
+        @isDisabled={{@isDisabled}}
+        @isReadOnly={{@isReadOnly}}
+        @rootTestSelector={{@rootTestSelector}}
+        name={{@name}}
+        ...attributes
+      >
+        <:label>{{yield to='label'}}</:label>
+        <:secondary as |secondary|>
```

Comment:
Does this need a `(has-block 'secondary')` wrapped around it?

Reply:
Sent a message about this, but we cannot conditionally render named blocks, so we have no way to do:

```hbs
<Form::Fields::Textarea
  @hint={{@hint}}
  @error={{this.mapErrors field.rawErrors}}
  @value={{this.assertString field.value}}
  {{! @glint-expect-error }}
  @onChange={{field.setValue}}
  @isDisabled={{@isDisabled}}
  @isReadOnly={{@isReadOnly}}
  @rootTestSelector={{@rootTestSelector}}
  name={{@name}}
  ...attributes
>
  <:label>{{yield to='label'}}</:label>
  
  {{#if (has-block "secondary")}}
    <:secondary as |secondary|>
  {{/if}}
```

So due to that we have the top-level `{{#if (has-block 'secondary')}}`